### ISSUE TITLE
Pull access denied for indrz/gogse

### ISSUE THREAD

**Issue by bouskdav** [indrz/indrz#129]
Hi,

i'm trying to make a fresh build with "make build" - with following result:

root@gzwiki:/home/administrator/indrz# make build-indrz
docker-compose build --build-arg ENV_TYPE= indrz_api
Building indrz_api
Sending build context to Docker daemon  462.3kB
Step 1/11 : FROM indrz/gogse:latest
**pull access denied for indrz/gogse, repository does not exist or may require 'docker login': denied: requested access to the resource is denied**
ERROR: Service 'indrz_api' failed to build : Build failed
make: *** [Makefile:21: build-indrz] Error 1

What am I doing wrong?

Thanks!
David

**Comment by mdiener21** [comment]
Hi David, sorry we are currently totally re-writing the deploy and setup pipelines.  I will also update the readme.md and documentation.  This should then solve this issue.

**Comment by ErmekZhumagulov** [comment]
I have almost the same problem:
Building indrz_api
Step 1/11 : FROM indrz/gogse:latest
ERROR: Service 'indrz_api' failed to build: pull access denied for indrz/gogse, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
Is that problem solved?

**Comment by mdiener21** [comment]
yes, we removed gogse as  a dependency, now simply use the Dockerfile in 
/devops/docker/local/indrz_api/Dockerfile    or use the docker-compose.local.yml
